ui/pages/confirmations/hooks/useCurrentConfirmation.ts

```diff
@@ -72,18 +85,28 @@ const useCurrentConfirmation = () => {
     (isRedesignedSignaturesUserSettingEnabled && isCorrectApprovalType) ||
     (isRedesignedConfirmationsDeveloperSettingEnabled && isCorrectApprovalType);
 
+  ///: BEGIN:ONLY_INCLUDE_IF(build-main,build-beta,build-flask)
   const shouldUseRedesignForTransactions =
```

Comment:
Rather than adding all these code fences and additional complexity to this file, could we instead just export an `IS_MMI` constant from another file (which uses a single code fence) and just update this condition to simply be:

```
const shouldUseRedesignForTransactions = !IS_MMI && ...
```

Reply:
Thanks @matthewwalsh0 I agree it would be simpler with less noise not having the code fences, let me look into it and see if we can change the approach 👍🏼

Reply:
I see your point also, but this feels like a special case as we know this is only temporary as eventually the legacy confirmations will be removed.